docs/v1/P4Runtime-Spec.adoc

```diff
@@ -6032,9 +6032,16 @@ error {
 The `Capabilities` RPC offers a mechanism through which a P4Runtime client can
 discover the capabilities of the P4Runtime server implementation. At the moment,
 the `CapabilitiesRequest` message is empty and the `CapabilitiesResponse`
-message only includes the `p4runtime_api_version` string field. This field must
-be set to the full semantic version string cite:[SemVer] corresponding to the
-version of the P4Runtime API implemented by the server, e.g. "1.1.0-rc.1".
+message includes the `p4runtime_api_version` string field and the `experimental`
```

Comment:
Nit: The current phrasing is a bit akward. Maybe "the `expirmental` field of type `Any`?